doc/Syscalls.md

```diff
@@ -94,14 +112,15 @@ The Yield syscall takes no arguments.
 
 ### 1: Subscribe
 
-Subscribe assigns callback functions to be executed in response to various events.
+Subscribe assigns callback functions to be executed in response to various
+events.
 
 The Subscribe syscall takes two arguments:
 
  - `subscribe_number`: An integer index for which function is being subscribed
  - `callback`: A pointer to a callback function to be executed when this event
  occurs. All callbacks conform to the C-style function signature:
- `void callback(int arg1, int arg2, int arg3, void* data)`
+ `void callback(int arg1, int arg2, int arg3, void* data)`.
```

Comment:
Again think this period is weird, particularly b/c this isn't a sentence.

If you're really attached to the period, you should probably also add one to the line above (though I'd prefer neither)